src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const getSum = (num) => {
+    if (num !== undefined) {
+      sum += num;
+
+      return getSum;
+    } else {
+      const result = sum;
+
```

Comment:
```suggestion
    } 
      const result = sum;

```

no need else statement after return